src/requestManager.cpp

```diff
@@ -102,15 +110,11 @@ void CRequestManager::AskFor(const CInv& obj, CNode* from, int priority)
 	  pendingTxns+=1;
 	  // all other fields are zeroed on creation
 	}
-      else  // existing
-	{
-	}
-
```

Comment:
IMO if that's the case we should leave them in but comment them out with
a short explanation such as // else - nothing to do 

On 04/09/2016 3:32 AM, gandrewstone wrote:
>
> In src/requestManager.cpp
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79#discussion_r77450168>:
>
> > @@ -102,15 +110,11 @@ void CRequestManager::AskFor(const CInv& obj, CNode* from, int priority)
> >  	  pendingTxns+=1;
> >  	  // all other fields are zeroed on creation
> >  	}
> > -      else  // existing
> > -	{
> > -	}
> > -
>
> This is my shorthand for "I have considered this case and there's
> nothing to do"...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/79/files/efaaa2474cc017ebc1c84fa3748b50c293bff5c7#r77450168>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMRF0Lf9AtPRbO6UUzFAKKJQLWPtSE30ks5qmp5agaJpZM4JokbP>.
>